bigquery/snippets/snippet_test.go

```diff
@@ -48,6 +48,8 @@ func TestAll(t *testing.T) {
 	tc := testutil.SystemTest(t)
 	ctx := context.Background()
 
+	buf := &bytes.Buffer{}
```

Comment:
Maybe move this below, closer to where it's used?

Reply:
Done.